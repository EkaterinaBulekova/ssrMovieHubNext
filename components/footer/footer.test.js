import React from 'react';
import { shallow } from 'enzyme';
import AppFooter from './footer';
import toJson from 'enzyme-to-json';

describe('<Footer />', () => {
  it('should render footer', () => {
    const component = shallow(<AppFooter/>);
    expect(toJson(component)).toMatchSnapshot();
  });

});