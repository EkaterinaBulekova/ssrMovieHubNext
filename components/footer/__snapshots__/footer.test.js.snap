// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> should render footer 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
